app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Container from "./Note/Container";
+import Header from './Note/Header';
+import Footer from './Note/Footer';
+import Content from './Note/Content';
+
+const Note = ({note, onClickEdit, onClickDelete}) => {
+  return (
```

Comment:
Perform implicit return here since there aren't any other side effects in this scope.